pkgs/models/db.go

```diff
@@ -0,0 +1,28 @@
+package models
+
+import (
+	"context"
+)
+
+type UsersDatabase interface {
+	ChangeRole(ctx context.Context, userID int, role string) (err error)
+	GetUserProfile(ctx context.Context, userID int, userEmail string) (res *UserProfile, err error)
+	GetUsersList(ctx context.Context, offset, limit int, sort, order string) (res *UsersListResponse, err error)
+	ChangeUserStatus(ctx context.Context, userID int, status bool) (err error)
+}
+
+type UserCredentials struct {
+	Username string
+	Password string
+}
+
+type UserRole struct {
+	Name string
+	Mask int64
+}
+
+type UsersDb struct {
```

Comment:
This struct contains all stub data and doesn't implement UserDatabase.
Please call this struct UserDbStub, hide it behind UserDatabase interface and do not use directly - use only UserDatabase interface.